lib/controllers/users.js

```diff
@@ -31,11 +31,11 @@ module.exports = Router()
     }
   })
   // TODO: This route should only be accessible to signed in users
-  .get('/me', (req, res) => {
+  .get('/me', authenticate, (req, res) => {
     res.json(req.user);
   })
   // TODO: This route should only be accessible to the admin user
-  .get('/', async (req, res, next) => {
+  .get('/', [authenticate, authorize], async (req, res, next) => {
```

Comment:
looks like this worked, but fyi you don't need to pass them as an array (express knows that all parameters between the first and last one are middlewear)